hledger-lib/Hledger/Reports/MultiBalanceReport.hs

```diff
@@ -25,6 +25,15 @@ module Hledger.Reports.MultiBalanceReport (
   sortRows,
   sortRowsLike,
 
+  -- * Helper functions
+  calculateReportSpan,
+  makeReportQuery,
+  getPostingsByColumn,
+  getPostings,
+  calculateColSpans,
+  startingBalances,
+  generateMultiBalanceReport,
+
```

Comment:
These exports don't seem to be needed by hledger.. was it for some local code ?

Reply:
Yes, if anybody wants to write their own report functions (which I have done occasionally), it's helpful to have access to them.